bower.json

```diff
@@ -2,19 +2,19 @@
   "name": "ember-cli-materialize",
   "dependencies": {
     "blanket": "~1.1.5",
-    "ember": "1.13.2",
+    "ember": "v2.0.3",
```

Comment:
Prefer to leave the demo app on ember 1.13, to ensure we retain support for Ember 1.x functionality